### ISSUE TITLE
Trsh Orca auxilliary basis demand

### ISSUE THREAD

**Issue by Lilachn91** [ReactionMechanismGenerator/ARC#665]
**Describe the bug**
ARC tried to trsh the following error in Orca "For DLPNO calculations we need at least one auxiliary basis set" using method 'change_node'.

**How to reproduce**
Orca input and output file attached, need to add to auxiliary basis '/c' at the end.
[input.txt](https://github.com/ReactionMechanismGenerator/ARC/files/11623145/input.txt)
[output.txt](https://github.com/ReactionMechanismGenerator/ARC/files/11623143/output.txt)

**Expected behavior**
ARC should alert the user of a problematic basis set, and then gracefully terminate.

**Comment by kfir4444** [comment]
thanks @Lilachn91!
Why should we terminate and not choose an appropriate basis set for the user?

**Comment by alongd** [comment]
@Lilachn91, is this still relevant?

**Comment by Lilachn91** [comment]
Yes, it still fails to troubleshoot when the user doesn't supply the auxiliary basis set properly, see the input and arc log files:

[arc.log](https://github.com/user-attachments/files/27778504/arc.log)
[input.txt](https://github.com/user-attachments/files/27778505/input.txt)

I'll add a fix to detect that and submit a new job.

**Comment by kfir4444** [comment]
Still reproduces on `main` @ `a24e13b0` (ORCA 6, PBS cluster), and I think the scope is broader than a "problematic basis set" — it affects **every** DLPNO level specified in the usual `method/basis` form, not a particular basis.

`Level` never derives an auxiliary basis, for any basis set:

```python
from arc.level import Level
for basis in ['def2-tzvp', 'cc-pvtz', 'def2-tzvpp', 'cc-pvdz', 'aug-cc-pvtz']:
    print(basis, Level(method='dlpno-ccsd(t)', basis=basis).auxiliary_basis)
# def2-tzvp  None
# cc-pvtz    None
# def2-tzvpp None
# cc-pvdz    None
# aug-cc-pvtz None
```

So a perfectly ordinary input:

```yaml
sp_level: dlpno-ccsd(t)/cc-pvtz
```

produces

```
!uHF dlpno-ccsd(t) cc-pvtz  tightscf normalpno
```

and ORCA stops before doing any work:

```
WARNING: For DLPNO calculations we need at least one auxiliary basis set
  ===> : Skipping actual calculation
[file orca_mdci/mdci_check.cpp, line 382]: Error (ORCA_MAIN): ... aborting the run
```

**What makes it costly is the failure mode rather than the error itself.** ARC treats it as a node problem, retries with `change_node`, then gives up, and the species is reported as unconverged:

```
Error: Could not troubleshoot sp for <label>! Tried troubleshooting 1 time(s), with the following methods: ['change_node'];
```

In my case the geometry optimization and frequency jobs had already succeeded, so the run looked like a failed TS search rather than a bad single-point level. Nothing in the ARC log mentions the auxiliary basis, so the only way to find it is to open the ORCA log by hand.

The workaround is to state it explicitly, which works fine:

```yaml
sp_level:
  method: dlpno-ccsd(t)
  basis: def2-tzvp
  auxiliary_basis: def2-tzvp/c
```

Two suggestions, either of which would have saved the round trip:

1. Default `auxiliary_basis` to `<basis>/C` when the method is a DLPNO variant and none is given — ORCA accepts `/C` for the usual Karlsruhe and Dunning sets.
2. Failing that, validate at input-parsing time and refuse to start, rather than discovering it after the geometry jobs have run. `change_node` cannot fix an input-level problem, so it would also be worth not attempting it for this error signature.
